public/css/icinga/widgets.less

```diff
@@ -224,6 +224,8 @@ table.multiselect tr[href] td {
 form.role-form {
   &.icinga-form .control-label-group {
     width: 20em;
+    // make labels wrap in Safari
+    line-break: anywhere;
```

Comment:
I don't like the use of `line-break` here. According to [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/line-break) this is a locale specific property which just happens to work in our case. What I specifically don't like is the statement `"disregarding any prohibition against line breaks"` which several parts of the permission name intentionally have and hence why this issue in Safari popped up to begin with.

I can't find a case where a `white-space: nowrap` isn't respected though, not with Chromium or Firefox. So this may only be a case of mis-interepretation or a different browser behavior despite what MDN says.

Is there a chance that another property has the same effect? Is MDN indeed wrong and the spec/browser behavior is entirely different? (Source pls)